<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NomenclatureComboBoxPopup</class>
 <widget class="QWidget" name="NomenclatureComboBoxPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabNomenclature">
      <attribute name="title">
       <string>&amp;Номенклатура</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="CTableView" name="tblNomenclature"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSearch">
      <attribute name="title">
       <string>&amp;Поиск</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblClass">
         <property name="text">
          <string>&amp;Класс</string>
         </property>
         <property name="buddy">
          <cstring>cmbClass</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="CRBComboBox" name="cmbClass"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblKind">
         <property name="text">
          <string>&amp;Вид</string>
         </property>
         <property name="buddy">
          <cstring>cmbKind</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="CRBComboBox" name="cmbKind"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblType">
         <property name="text">
          <string>&amp;Тип</string>
         </property>
         <property name="buddy">
          <cstring>cmbType</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="CRBComboBox" name="cmbType"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblCode">
         <property name="text">
          <string>&amp;Код</string>
         </property>
         <property name="buddy">
          <cstring>edtCode</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="edtCode"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblName">
         <property name="text">
          <string>&amp;Наименование</string>
         </property>
         <property name="buddy">
          <cstring>edtName</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QCheckBox" name="chkIncludeAnalogies">
         <property name="text">
          <string>Включая аналоги</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>178</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="2">
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLineEdit" name="edtProducer"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblProducer">
         <property name="text">
          <string>&amp;Производитель</string>
         </property>
         <property name="buddy">
          <cstring>edtProducer</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblATC">
         <property name="text">
          <string>Код &amp;АТХ</string>
         </property>
         <property name="buddy">
          <cstring>edtATC</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QLineEdit" name="edtATC"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>tblNomenclature</tabstop>
  <tabstop>cmbClass</tabstop>
  <tabstop>cmbKind</tabstop>
  <tabstop>cmbType</tabstop>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtProducer</tabstop>
  <tabstop>edtATC</tabstop>
  <tabstop>chkIncludeAnalogies</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
